conf/modules.config

```diff
@@ -542,6 +542,21 @@ process {
         ]
     }
 
+    withName: TAXPASTA_MERGE {
+        ext.args =  {
+            [
+                "-p ${meta.tool} -o ${meta.tool}_${meta.id}.${params.taxpasta_standardisation_format}",
+                params.taxpasta_add_taxonomy ? "-p" : "",
+                params.taxpasta_add_samplesheet ? "-s" :""
```

Comment:
I don't think we need to support this here, unless we want to use it to customise samplenames (i.e., we build a samplesheet via a reader function...) thoughts @Midnighter ? How important do you think this would be

Reply:
Those arguments could be passed by `ext.args` instead?

Reply:
no, they have to be via input channels because they are files and need to be staged.

Taxonomy should be kept, but I'm not sure if exposing the samplesheet input to the user makes sense.

Reply:
Then maybe we should have taxonomy via input channels as well, as it is a path to taxdump files?

Reply:
Then, I will remove the samplesheet and keep the taxonomy if I understood it correctly.

Reply:
Correct!